vdm-io/Joomla-Component-Builder
# Transliteration of cyrillic characters.

Pull Request for Issue gh- .

### Summary of Changes
- Transliteration of cyrillic names of Views when preparing safe string

### Testing Instructions
- Make Admin view named in russian like "Таблица"
- Add this view to component
- Compile component
- Check folder admin/view

### Expected result
- Folders in admin/views:
    - tablica
    - tablici

### Actual result
- No folders of this view in admin/views


### Documentation Changes Required
No changes required.
